resource_reaper.py

```diff
@@ -0,0 +1,147 @@
+'''
+Automated Vertex AI resource cleanup for python-docs-samples-test
+'''
+
+# ! pip3 install google-cloud-aiplatform google-cloud-bigquery --quiet
+
+from google.cloud import aiplatform
+from google.cloud import bigquery
+import time
+import datetime
+
+PROJECT_ID = "python-docs-samples-tests"  # @param {type:"string"}
+
+# Set the project id
+# ! gcloud config set project {PROJECT_ID}
```

Comment:
suggestion: remove these comments. The CI/CD should already have the project set. If a maintainer uses this script, they should _hopefully_ know to set the project.